hurricane-hazard-import-tool.py

```diff
@@ -7,7 +7,7 @@
             print('Python 2 is default interpreter...going to try Python 3')
             py_check = check_output('where python').split('\r\n')
             py3_intepreter = ''.join([i for i in py_check if 'miniforge' in i])
-            the_call = py3_intepreter + ' ' + sys.argv[0]
+            the_call = py3_intepreter + ' hurricane-hazard-import-tool.py'
```

Comment:
Fortify still showed the vulnerability when I ran it locally. It's not liking the use of the call function.